src/daft-local-execution/src/intermediate_ops/mod.rs

```diff
@@ -1,3 +1,4 @@
+pub mod actor_pool_project;
```

Comment:
Nit: Would it be possible to `#[cfg(feature = "python")]` at the module instead of inside? I'm wondering if we can consolidate the `#[cfg(feature = "python")] / #[cfg(not(feature = "python"))]` logic to a single location, perhaps in the `physical_plan_to_pipeline` translation.

Reply:
I did `#[cfg(feature = "python")]` in here so that it would be possible to write Rust tests for the actor pool project pipeline node. I don't have any right now but might add some in the future

Reply:
^ I did find it very useful to do this, because you might imagine cases where you want the pipeline node to be available (even when not compiling for Python) to do things such as testing optimization passes on the pipeline nodes